bitmap.js

```diff
@@ -0,0 +1,39 @@
+'use strict'; 
+
+const fileHelper = require('./lib/bitmap-file-helper.js');
+const Bitmap = require('./model/bitmap-constructor.js');
+
+
+let filesToConvert = [`${__dirname}/img/palette-bitmap.bmp`, `${__dirname}/img/optimus-prime.bmp`];
+transformBMP(filesToConvert);
+
+function transformBMP(files) {
+  let colorTransforms = [];
+  if (process.argv.length > 2) {
```

Comment:
@brials @remilonwheels @caylazabel nice use of command line arguments and an associated catch to see if they exist